src/Test/_project.json

```diff
@@ -37,20 +37,37 @@
         "xunit.console.netcore": "1.0.2-prerelease-00101",
         "xunit.netcore.extensions": "1.0.0-prerelease-00153",
         "xunit.runner.utility": "2.1.0",
-        "Agent.Listener": { "target": "project", "version": "1.0.0-*" },
+        "Agent.Listener": {
+            "target": "project",
+            "version": "1.0.0-*"
+        },
         "vss-api-netcore": "0.5.6-private",
-        "Microsoft.VisualStudio.Services.Agent": { "target": "project", "version": "1.0.0-*" },
-        "Agent.Worker": { "target": "project", "version": "1.0.0-*" } 
+        "Microsoft.VisualStudio.Services.Agent": {
+            "target": "project",
```

Comment:
IMO, the one line for dependency looked better.. I could scan the dependency on the left hand side in a single parse. Now its little bit hard to see what dependencies are required. @bryanmacfarlane @ericsciple any comments?